cmd/dumpling/main.go

```diff
@@ -122,6 +123,7 @@ func main() {
 	pflag.StringVar(&csvSeparator, "csv-separator", ",", "The separator for csv files, default ','")
 	pflag.StringVar(&csvDelimiter, "csv-delimiter", "\"", "The delimiter for values in csv files, default '\"'")
 	pflag.StringVar(&outputFilenameFormat, "output-filename-template", "", "The output filename template (without file extension), default '{{.DB}}.{{.Table}}.{{.Index}}'")
+	pflag.BoolVar(&chunkByRegion, "chunk-by-region", false, "whether to use region info to split table chunks")
```

Comment:
how about turning on the `chunk-by-region` by default when the upstream database is TiDB?

Reply:
I don't think so. table-regions count are usually very big (like 300000 per region). It may confuse users if they set `rows` to a small number.

Reply:
fine